uiplib/gui/settingsTab.py

```diff
@@ -175,7 +176,8 @@ def handle_settings(appObj):
                                "Invalid value for no of images")
         return
     try:
-        appObj.new_settings['days-to-autodel'] = int(appObj.auto_del_val.get())
+        val = int(appObj.auto_del_val.get())*24*60*60
```

Comment:
Remove the integer parsing in other file, since you're already using try except here.